documentation/iotc-azurertos-sdk/samples/ck-rx65n/DEVELOPER_GUIDE.md

```diff
@@ -202,6 +205,9 @@ will suggest a download.
 
 ## Clean and build the project
 
+***IMPORTANT:*** Though Blueboard got working fairly quickly it may be the case you'd need to tweak generated code manually.
```

Comment:
Outdated comment. New changes to ck-rx65n project were introduced and it should run fine out of the box now